lib/ckb/always_success_wallet.rb

```diff
@@ -79,29 +78,37 @@ def install_mruby_cell!(mruby_cell_filename)
 
     def configuration_installed?(configuration)
       cell_with_status = api.get_live_cell(configuration[:out_point])
-      return false if cell_with_status[:status] != "live"
-      returned_cell_hash = Ckb::Utils.bin_to_prefix_hex(
-        Ckb::Blake2b.digest(Ckb::Utils.hex_to_bin(cell_with_status[:cell][:data])))
+      return false if cell_with_status[:status] != 'live'
+
+      returned_cell_hash = Ckb::Utils.bin_to_hex(
+        Ckb::Blake2b.digest(
+          Ckb::Utils.hex_to_bin(
+            cell_with_status[:cell][:data]
+          )
+        )
+      )
       unless returned_cell_hash == configuration[:cell_hash]
         raise "Cell hash doesn't match, something weird is happening!"
       end
+
       true
     end
 
     def get_balance
-      get_unspent_cells.map { |c| c[:capacity] }.reduce(0, &:+)
+      get_unspent_cells.map { |c| c[:capacity].to_i }.reduce(0, &:+)
```

Comment:
```suggestion
      get_unspent_cells.map { |c| c[:capacity].to_i }.reduce(:+)
```

Reply:
Without default `0`, empty array will return nil, it may cause some problems.

Reply:
I see. 👍